e2e-tests/scheduled-backup/run

```diff
@@ -68,18 +68,18 @@ start_minio() {
         --set environment.MINIO_REGION=us-east-1 \
         --set environment.MINIO_HTTP_TRACE=/tmp/trace.log \
         stable/minio
-    MINIO_POD=$(kubectl get pods --selector=release=minio-service -o 'jsonpath={.items[].metadata.name}')
+    MINIO_POD=$(kubectl_bin get pods --selector=release=minio-service -o 'jsonpath={.items[].metadata.name}')
```

Comment:
function run command few times, so function output can have artifacts in output from all runs (not only the last)

Reply:
fixed